charts/4allportal/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: "3.10.35"
 description: A Helm chart for 4ALLPORTAL version 3.10.0 and up
 name: 4allportal
-version: 19.0.42
+version: 19.0.43
```

Comment:
you haven't changed anything in the 4allportal chart so no version bump necessary